core-client/tests/integration_test.rs

```diff
@@ -778,26 +769,14 @@ async fn test_centralized_custodian_backup(ctx: &DockerComposeCentralizedCustodi
     // end points, and content returned from the KMS to the custodians, work as expected.
 }
 
-#[ignore]
-#[test_context(DockerComposeThresholdContextDefault)]
-#[tokio::test]
-#[serial(docker)]
-async fn test_threshold_secure(ctx: &mut DockerComposeThresholdContextDefault) {
-    init_testing();
-    let temp_dir = tempfile::tempdir().unwrap();
-    let keys_folder = temp_dir.path();
-    let (key_id, _crs_id) = key_and_crs_gen(ctx, keys_folder, false).await;
-    integration_test_commands(ctx, key_id).await;
-}
-
 #[test_context(DockerComposeThresholdContextDefault)]
 #[tokio::test]
 #[serial(docker)]
 async fn test_threshold_insecure(ctx: &mut DockerComposeThresholdContextDefault) {
```

Comment:
Not related to this PR but could could you add some documentation on the fact that threshold tests must be named `threshold`, centralized `centralized` and nightly `nightly`?

Reply:
done in https://github.com/zama-ai/kms/pull/176/commits/7b7696a92108d21b34dfd1b73f25e4704cf85a06